README.md

```diff
@@ -91,6 +91,10 @@ docker-compose up -d
 ```bash
 curl -X GET "<YOUR HOST>/webhook?hub.verify_token=<YOUR VERIFY TOKEN>&hub.chalenge=CHALLENGE_ACCEPTED&hub.mode=subscribe&init_bot=true"
 ```
+If your webhook verification is working as expected, you should see the following:
+
+-```WEBHOOK_VERIFIED``` logged to the command line where your node process is running.\
```

Comment:
please implement it in [messenger.py](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/9493a26b6ab7851bf2d49d59cb11af5cbe27d741/src/services/messenger.py)

Reply:
I added instructions based on issue #25, as I understand the issue, the proposed changes are for the README part I guess.

Reply:
You're right @Zenix27! LGTM and it good to go. Thank you for taking time fixing those issues :)

Reply:
Sure Thanks, @aboullaite @elinguiuriel